### PR TITLE
Replace CSS variables with Sass variables

### PR DESCRIPTION
**Pull request by skoolbus39**
- Update default color variables
- Replace CSS variables with Sass variables for greater flexibility. (CSS variables can still be set at the theme level, but sites can also use static values [e.g., hex, RGB, or HSL colors] if desired for some or all values.)

### CODE REVIEW

File: scss/components/_components.forms-selects.scss [hunk 1]
```diff
@@ -50,12 +50,12 @@
...
 .dcf-form select:hover {
-  border-color: var(--b-input-hover);
+  border-color: $border-color-input-hover;
 }
 
 
 .dcf-input-select:focus, // TODO: deprecate .dcf-input-select?
 .dcf-form select:focus {
-  border-color: var(--b-input-focus);
+  border-color: $border-color-input-focus);
```
Comment (jsturek) [inline scss/components/_components.forms-selects.scss]: Typo need to remove `)` so it's `border-color: $border-color-input-focus;`

**Comment by jsturek** [comment]
Still see `var(` references in `_mixins.colors.scss` (5 times), `_components.nav-toggle.scss` (2 times), `_mixins.backgrounds.scss` (3 times)

**Comment by jsturek** [comment]
Are these going way in WDN as well as I made numerous changes to apps for dark mode to look for these css variables?

**Comment by skoolbus39** [comment]
To answer @jsturek's question: they're not going away in [wdntemplates](https://github.com/unl/wdntemplates); they're just getting moved into the variables files. Also, I'm merging this now to test with [dcf_starter](https://github.com/digitalcampusframework/dcf_starter) and wdntemplates, and will follow-up with any additional edits if needed.

**Pull request merged**